src/components/docker/IconPickerDialog.tsx

```
@@ -59,58 +73,73 @@ export default function IconPickerDialog({
+                      height: virtualRow.size,
+                      transform: `translateY(${virtualRow.start}px)`,
+                    }}
+                  >
+                    {row.map((slug) => (
+                      <button
+                        key={slug}
+                        onClick={() => handleSelect(slug)}
+                        className={`flex flex-col items-center p-2 rounded-md transition-colors hover:bg-blue-500/10 ${
+                          currentIcon === slug ? 'bg-blue-500/20 ring-1 ring-blue-500' : ''
+                        }`}
+                      >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)